ContextDesktop/src/test/java/com/github/migi_1/Context/model/TestMainEnvironment.java

```diff
@@ -146,8 +146,13 @@ public void setUp() throws Exception {
         Mockito.when(wrapper.getServer()).thenReturn(Mockito.mock(Server.class));
         PowerMockito.whenNew(Platform.class).withAnyArguments().thenReturn(platform);
         PowerMockito.whenNew(CarrierAssigner.class).withAnyArguments().thenReturn(carrierAssigner);
+<<<<<<< HEAD
+        PowerMockito.suppress(PowerMockito.method(MainEnvironment.class, "initLights"));
+=======
         PowerMockito.whenNew(EnemySpawner.class).withAnyArguments().thenReturn(enemySpawner);
+>>>>>>> origin/master
         env = PowerMockito.spy(new MainEnvironment(Mockito.mock(CarrierAssigner.class)));
+        //env.setApp(app);
```

Comment:
In the merge with master you left a merge conflict in the code. This is why the travis build fails. Please fix this